extensions/mssql/src/ui/dialogBase.ts

```diff
@@ -147,6 +147,17 @@ export abstract class DialogBase<DialogResult> {
 		return this.createInputBox(ariaLabel, textChangeHandler, value, enabled, 'password', width);
 	}
 
+	protected createInputBoxWithProperties(ariaLabel: string, textChangeHandler: (newValue: string) => Promise<void>, properties: azdata.InputBoxProperties, customValidation?: () => Promise<boolean>): azdata.InputBoxComponent {
+		properties.ariaLabel = ariaLabel;
```

Comment:
Aria label is already in the properties, is there a reason you're having it be a separate property?